lib/resources/service.rb

```diff
@@ -134,9 +134,9 @@ def select_service_mgmt # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCom
         else
           SysV.new(inspec, service_ctl || '/usr/sbin/service')
         end
-      elsif %w{redhat fedora centos}.include?(platform)
+      elsif %w{redhat fedora centos parallels virtuozzo}.include?(platform)
```

Comment:
maybe we should add `parallels` and `virtuozzo` platforms to https://github.com/chef/train/blob/master/lib/train/extras/os_detect_linux.rb#L17 and use os.redhat? + `fedora` as defined in https://github.com/chef/train/blob/master/lib/train/extras/os_common.rb#L45 in here